homeassistant/components/qbittorrent/config_flow.py

```diff
@@ -55,3 +66,31 @@ async def async_step_user(
 
         schema = self.add_suggested_values_to_schema(USER_DATA_SCHEMA, user_input)
         return self.async_show_form(step_id="user", data_schema=schema, errors=errors)
+
+    @staticmethod
```

Comment:
I think we should not make this an configurable option but just create all entities, but disable a lot of them by default.

What do you think?

Reply:
hey @joostlek, if that's a more standard convention for HA components (rather than only creating entities conditionally), then that's ok with me. How would I set a configured entity to be disabled inside SensorEntity?

Reply:
Left an example suggestion